app/views/pages/rollover.html.erb

```diff
@@ -7,28 +7,41 @@
     </h1>
 
     <p class="govuk-body">
-      Courses for the next cycle (<%= next_recruitment_cycle_period_text %>) will be published on <%= Settings.next_cycle_open_date.to_s(:govuk) %>.
+      Your courses, locations and details have been copied.
     </p>
+
     <p class="govuk-body">
-      Your courses, locations and organisation details have been copied from the current cycle.
+      Courses for the next cycle will be published on <%= Settings.next_cycle_open_date.to_s(:govuk) %>.
     </p>
 
-    <h2 class="govuk-heading-m">What you need to do</h2>
-
     <p class="govuk-body">
-      Before you can publish courses for the next cycle, you'll need to confirm:
+      Before then, you can:
     </p>
 
     <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-4">
-      <li>degree requirements</li>
-      <li>GCSE requirements</li>
-      <li>if you can sponsor visas</li>
+      <li>add new courses</li>
+      <li>delete courses</li>
+      <li>edit the details of your courses</li>
     </ul>
 
+    <h2 class="govuk-heading-m">New for the <%= next_recruitment_cycle_period_text %> cycle</h2>
+
     <p class="govuk-body">
-      You can also add, edit or delete courses and locations.
+      All applications will be made via Apply for teaching training, not UCAS.
     </p>
 
+    <p class="govuk-body">
+      You will need to confirm:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-4">
+      <li>degree requirements</li>
+      <li>GCSE requirements</li>
+      <li>whether you can sponsor visas</li>
+      <li>locations of school placements</li>
+      <li>a URN for course locations and a UKPRN for the organisation</li>
```

Comment:
@gregknight1 I think this last bullet would read better as

- the urn for each school course location
- the UKPRN for your organisation

Reply:
@dankmitchell - Yes, Dan - agree with that.

Reply:
Do we need to mention 
- the URN for your organisation 
(given that 700 out of 1100) would need to add it...

Reply:
@defong I'm not really up to speed with Publish stuff but I thought URNs were for courses only?

Reply:
@gregknight1 I think UKPRN is on all providers and URN is on locations but URN is also on provider if the provider is a lead school (which is 784 providers out of around 1100)

Reply:
Thanks all. @darokel , final content -

- the URN for each school course location
- the UKPRN for your organisation

Conditional if user is associated to a provider where the `provider_type` is `lead_school`

- the URN for your organisation

Reply:
done https://github.com/DFE-Digital/publish-teacher-training/pull/1774/commits/b9336b7d4e4db58cd6e714d64b42acf4d4470a0b